public/components/mui-0.1.22-rc1/js/mui.min.js

```diff
@@ -0,0 +1 @@
:f.raiseError("Expecting 'on' or 'off'");return t}function o(e,t){var n=document.body,i=document.getElementById(p);if(h.addClass(n,m),i){for(;i.firstChild;)i.removeChild(i.firstChild);t&&i.appendChild(t)}else i=document.createElement("div"),i.setAttribute("id",p),t&&i.appendChild(t),n.appendChild(i);return v.test(navigator.userAgent)&&h.css(i,"cursor","pointer"),e.keyboard?s():l(),e["static"]?u(i):c(i),i.muiOptions=e,i}function r(){var e,t=document.getElementById(p);if(t){for(;t.firstChild;)t.removeChild(t.firstChild);t.parentNode.removeChild(t),e=t.muiOptions.onclose}return h.removeClass(document.body,m),l(),u(t),e&&e(),t}function s(){h.on(document,"keyup",a)}function l(){h.off(document,"keyup",a)}function a(e){27===e.keyCode&&r()}function c(e){h.on(e,"click",d)}function u(e){h.off(e,"click",d)}function d(e){e.target.id===p&&r()}var f=e("./lib/util.js"),h=e("./lib/jqLite.js"),m="mui-overlay-on",p="mui-overlay",v=/(iPad|iPhone|iPod)/g;t.exports=i},{"./lib/jqLite.js":5,"./lib/util.js":6}],9:[function(e,t,n){"use strict";function i(e){e._muiRipple!==!0&&(e._muiRipple=!0,"INPUT"!==e.tagName&&(r.on(e,"touchstart",o),r.on(e,"mousedown",o)))}function o(e){if(0===e.button){var t=this;if(t.disabled!==!0&&t.touchFlag!==!0){t.touchFlag=!0,setTimeout(function(){t.touchFlag=!1},100);var n=document.createElement("div");n.className=u;var i,o,s=r.offset(t),l=e.pageX-s.left,d=e.pageY-s.top;i=r.hasClass(t,c)||"fab"===t.getAttribute(a)?s.height/2:s.height,o=i/2,r.css(n,{height:i+"px",width:i+"px",top:d-o+"px",left:l-o+"px"}),t.appendChild(n),window.setTimeout(function(){t.removeChild(n)},2e3)}}}var r=e("./lib/jqLite.js"),s=e("./lib/util.js"),l="mui-btn",a="data-mui-style",c="mui-btn-floating",u="mui-ripple-effect";t.exports={initListeners:function(){for(var e=document,t=e.getElementsByClassName(l),n=t.length-1;n>=0;n--)i(t[n]);s.onNodeInserted(function(e){r.hasClass(e,l)&&i(e)})}}},{"./lib/jqLite.js":5,"./lib/util.js":6}],10:[function(e,t,n){"use strict";function i(e){e._muiTabs!==!0&&(e._muiTabs=!0,s.on(e,"click",o))}function o(e){if(0===e.button){var t=this;null===t.getAttribute("disabled")&&setTimeout(function(){e.defaultPrevented||r(t)},0)}}function r(e){var t,n,i,o,r=e.parentNode,a=e.getAttribute(u),c=document.getElementById(a);for(c||l.raiseError('Tab pane "'+a+'" not found'),t=r.parentNode.children,o=t.length-1;o>=0;o--)i=t[o],i!==r&&s.removeClass(i,d);for(n=c.parentNode.children,o=n.length-1;o>=0;o--)i=n[o],i!==c&&s.removeClass(i,d);s.addClass(r,d),s.addClass(c,d)}var s=e("./lib/jqLite.js"),l=e("./lib/util.js"),a="data-mui-toggle",c="["+a+'="tab"]',u="data-mui-controls",d="mui-is-active";t.exports={initListeners:function(){for(var e=document.querySelectorAll(c),t=e.length-1;t>=0;t--)i(e[t]);l.onNodeInserted(function(e){"tab"===e.getAttribute(a)&&i(e)})},api:{activate:function(e){var t="["+u+"="+e+"]",n=document.querySelectorAll(t);n.length||l.raiseError('Tab control for pane "'+e+'" not found'),r(n[0])}}}},{"./lib/jqLite.js":5,"./lib/util.js":6}]},{},[7]);
```

Comment:
Line is too long.<br>Expected '{' and instead saw 'return'.<br>Missing semicolon.<br>Expected an identifier and instead saw '='.<br>Expected an assignment or function call and instead saw an expression.<br>Expected '{' and instead saw 'o'.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>Confusing use of '!'.<br>Possible strict violation.<br>'n' is already defined.<br>'r' is already defined.<br>Mixed double and single quotes.<br>Expected '{' and instead saw 'i'.<br>Expected an identifier and instead saw '==='.<br>Too many errors. (100% scanned).